.github/workflows/release.yaml

```diff
@@ -0,0 +1,254 @@
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: release/setup-node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version-file: "package.json"
+          cache: "npm"
+          cache-dependency-path: package-lock.json
+      - name: release/optimize
+        shell: powershell
+        run: ./scripts/Makefile.ps1 optimize
+      - name: release/install-deps
+        shell: powershell
+        run: ./scripts/Makefile.ps1 install-deps
+      - name: release/test
+        uses: ./.github/actions/test
+      - name: release/build
+        shell: powershell
+        env:
+          MM_WIN_INSTALLERS: 1
+          PFX_KEY: ${{ secrets.MM_DESKTOP_MSI_INSTALLER_PFX_KEY }}
+          CSC_KEY_PASSWORD: ${{ secrets.MM_DESKTOP_MSI_INSTALLER_CSC_KEY_PASSWORD }}
+          PFX: ${{ secrets.MM_DESKTOP_MSI_INSTALLER_PFX }}
+          CSC_LINK: ${{ secrets.MM_DESKTOP_MSI_INSTALLER_CSC_LINK }}
+        run: |
+          ./scripts/Makefile.ps1 build
+      - name: release/package
+        run: |
+          mkdir -p ./build/win-release
+          bash -x ./scripts/patch_updater_yml.sh
+          bash -x ./scripts/cp_artifacts.sh release ./build/win-release
+      - name: release/upload-build
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        with:
+          name: build
+          path: ./build
+          retention-days: 14
+
+  build-mac-installer:
+    runs-on: macos-12
+    needs:
+      - mac-app-store-preflight
+    steps:
+      - name: release/checkout-repo
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: release/setup-node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version-file: "package.json"
+          cache: "npm"
+          cache-dependency-path: package-lock.json
+      - name: release/create-build-folder
+        run: mkdir -p ./build
+      - name: release/install-dependencies
+        run: |
+          brew install yq rename
+          npm ci
+      - name: release/test
+        uses: ./.github/actions/test
+      - name: release/build
+        env:
+          APPLEID: ${{ secrets.MM_DESKTOP_MAC_INSTALLER_APPLE_ID }}
+          APPLEIDPASS: ${{ secrets.MM_DESKTOP_MAC_INSTALLER_APPLE_ID_PASS }}
+          CSC_FOR_PULL_REQUEST: true
+          CSC_KEY_PASSWORD: ${{ secrets.MM_DESKTOP_MAC_INSTALLER_CSC_KEY_PASSWORD }}
+          CSC_LINK: ${{ secrets.MM_DESKTOP_MAC_INSTALLER_CSC_LINK }}
+        run: |
+          mkdir -p ./build/macos-release
+          npm run package:mac-with-universal
+          bash -x ./scripts/patch_updater_yml.sh
+          bash -x ./scripts/cp_artifacts.sh release ./build/macos-release
+      - name: release/rename-arm64-to-m1
+        run: rename 's/arm64/m1/' ./build/macos-release/$(jq -r .version package.json)/*
```

Comment:
Maybe decouple the .version extract with a variable, and use the Variable to inject this to the command. 
Would something like the following work: ? 
```
env:
  APP_VERSION: $(jq -r .version package.json))
```